zerver/migrations/0352_migrate_twenty_four_hour_time_to_realmuserdefault.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.6 on 2021-09-17 17:08
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def migrate_twenty_four_hour_time_to_realmuserdefault(
+    apps: StateApps, schema_editor: DatabaseSchemaEditor
+) -> None:
+    RealmUserDefault = apps.get_model("zerver", "RealmUserDefault")
+    realm_user_default_objects = RealmUserDefault.objects.all()
```

Comment:
We can make this much faster by doing this filter query:

`RealmUserDefault.objects.exclude(twenty_four_hour_time=F("realm__default_twenty_four_hour_time"))`